katcp/inspecting_client.py

```diff
@@ -109,12 +174,15 @@ class InspectingClientAsync(object):
     """
 
     sync_timeout = 5
```

Comment:
As discussed IRL the proper solution should be addressed in CB-1609. I think we should not go down the route of arbitrarily long timeouts since that tends to mask real problems while making a system lazy in the face of trouble :) 

As a compromise increased to 8s with a long TODO comment :)